frameworks/helloworld/src/main/dist/multiport.yml

```diff
@@ -0,0 +1,65 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  multiport:
+    count: {{HELLO_COUNT}}
+    resource-sets:
+      multi-port-resources:
+        ports:
+          port_one:
+            port: {{HELLO_PORT_ONE}}
+          port_two:
+            port: {{HELLO_PORT_TWO}}
+          port_three:
+            port: {{HELLO_PORT_THREE}}
+    tasks:
+      server:
+        goal: ONCE
+        resource-set: multi-port-resources
+        cmd: |
+          networktool=netcat
+          if hash ncat 2>/dev/null; then
+              networktool=ncat
+              echo "ncat is installed"
+          elif hash nc 2>/dev/null; then
+              networktool=nc
+              echo "nc is installed"
+          elif hash netcat 2>/dev/null; then
+              networktool=netcat
+              echo "netcat is installed"
+          else
+              echo "Networking tools are not installed. Exiting.."
+              exit 0
+          fi
+
+          echo "Using networking tool : ${networktool}"
+
+          ${networktool} -l {{HELLO_PORT_ONE}} &
+          pid_1=$!
+          ${networktool} -l {{HELLO_PORT_TWO}} &
+          pid_2=$!
+          ${networktool} -l {{HELLO_PORT_THREE}} &
+          pid_3=$!
+
+          $(${networktool} -v multiport-0-server.$FRAMEWORK_HOST {{HELLO_PORT_ONE}})
+          exit_1=$?
+          $(${networktool} -v multiport-0-server.$FRAMEWORK_HOST {{HELLO_PORT_TWO}})
```

Comment:
We could use `$TASK_NAME` instead of `multiport-0-server` here which would make it work for multiple pods too.